core/src/lib/tls_openssl.cc

```diff
@@ -139,7 +139,6 @@ bool TlsOpenSsl::TlsPostconnectVerifyCn(
     char data[256]; /* nullterminated by X509_NAME_get_text_by_NID */
     if (X509_NAME_get_text_by_NID(subject, NID_commonName, data, sizeof(data))
         > 0) {
-      std::string cn;
       for (const std::string& cn : verify_list) {
         std::string d(data);
```

Comment:
yes. probably doesn't make sense to re-initialize `d` every time. I'd make it a `const std::string` though.

Reply:
Shouldnt make a big difference.  I proposed string_view to prevent an allocation but in practice this should not matter.  Take care about the dmsg right after this as well.  We probably want to change it to data.c_str() there if you use std::string (with std::string_view this also works, but you have to change the format specifier accordingly).